airflow/www/views.py

```diff
@@ -531,7 +534,8 @@ def get_logs_with_metadata(self, session=None):
                     execution_date))
             response = jsonify({'error': error_message})
             response.status_code = 400
-
+            # Bad format, don't count as fetch failure.
+            Stats.decr('webserver.get_logs_with_metadata.failures')
```

Comment:
```suggestion
            Stats.incr('webserver.get_logs_with_metadata.failures')
```
Why is this metric decreased?